environments/staging/staging-platform-hmcts-net.yml

```diff
@@ -34,6 +34,9 @@ A:
     - 10.50.10.117
     ttl: 300
 cname:
+  - name: c100-application
+    ttl: 300
+    record: sdshmcts-stg.azurefd.net
```

Comment:
no need to move this it was already right?

Reply:
I had added it, later saw the dupe and deleted the other one. So it looks moved but wasn't the intention.